plexapi/video.py

```diff
@@ -207,9 +211,18 @@ def episodes(self, watched=None):
         leavesKey = '/library/metadata/%s/allLeaves' % self.ratingKey
         return utils.listItems(self.server, leavesKey, watched=watched)
 
-    def episode(self, title):
-        path = '/library/metadata/%s/allLeaves' % self.ratingKey
-        return utils.findItem(self.server, path, title)
+    def episode(self, title=None, season=None, episode=None):
```

Comment:
I like the idea of title or season & episode. We should raise an exception at the end of this function if we fall through the if statements explaining that they need to specify both season & episode for the second case to work.